src/SprykerSdk/Spryk/Model/Spryk/Filter/TypedArrayFilter.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Filter;
+
+class TypedArrayFilter implements FilterInterface
+{
+    protected const FILTER_NAME = 'typedArrayConvert';
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return static::FILTER_NAME;
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return string
+     */
+    public function filter(string $value): string
+    {
+        $initialParameters = preg_split('/(\s*,*\s*)*,+(\s*,*\s*)*/', $value);
+        $filteredParameters = [];
+
+        foreach ($initialParameters as $parameter) {
+            $parameterExploded = explode(' ', $parameter);
```

Comment:
I was thinking that maybe split by space with limit of 1 will give you two parts always, maybe this is more convenient in terms of readability: it will be more obvious that you want to get the type and the variable name.

Reply:
Great idea, thanks! Fixed it.